library/src/test/java/com/chuckerteam/chucker/internal/support/JsonConverterTest.kt

```diff
@@ -1,60 +1,14 @@
 package com.chuckerteam.chucker.internal.support
 
 import com.google.common.truth.Truth.assertThat
-import java.text.DateFormat
-import java.util.Date
-import java.util.Locale
-import org.junit.jupiter.api.Test
+import org.junit.Test
 
 class JsonConverterTest {
 
     @Test
     fun testInstance_sameInstance() {
         val instance1 = JsonConverter.instance
         val instance2 = JsonConverter.instance
-
         assertThat(instance1).isEqualTo(instance2)
     }
-
-    @Test
-    fun testGsonConfiguration_willParseDateTime() {
-        val dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
-        val expectedDateString = dateFormat.format(Date(0))
-        val json = JsonConverter.instance.toJson(DateTestClass(Date(0)))
-        assertThat(json).isEqualTo(
-            """
-            {
-              "date": "$expectedDateString"
-            }
-            """.trimIndent()
-        )
-    }
-
-    @Test
-    fun testGsonConfiguration_willUseLowerCaseWithUnderscores() {
-        val json = JsonConverter.instance.toJson(NamingTestClass("aCamelCaseString"))
```

Comment:
Can I ask you to save this test?

Reply:
Moshi doesn't support field naming policy.
It recommends us to name the field as it appears in the JSON.
More here: https://github.com/square/moshi#borrows-from-gson

Also, nowhere in the codebase, we were using this feature provided by Gson.
We are only serializing and deserializing `List<HttpHeader>` which is already annotated with `@Json`.

Reply:
++ we can drop this comment then